lib/Data/Mapper/UserRegisterMapper.php

```diff
@@ -7,78 +7,12 @@
  */
 namespace EzSystems\RepositoryForms\Data\Mapper;
 
-use eZ\Publish\API\Repository\Values\Content\Field;
-use EzSystems\RepositoryForms\Data\Content\FieldData;
-use EzSystems\RepositoryForms\Data\User\UserRegisterData;
-use EzSystems\RepositoryForms\User\RegistrationContentTypeLoader;
-use EzSystems\RepositoryForms\User\RegistrationGroupLoader;
-use Symfony\Component\OptionsResolver\OptionsResolver;
+use EzSystems\EzPlatformUser\Form\DataMapper\UserRegisterMapper as BaseUserRegisterMapper;
 
 /**
  * Form data mapper for user creation.
+ * @deprecated Deprecated in 1.1 and will be removed in 2.0. Please use EzSystems\EzPlatformUser\Form\DataMapper\UserRegisterMapper instead.
  */
-class UserRegisterMapper
+class UserRegisterMapper extends BaseUserRegisterMapper
```

Comment:
By using inheritance this class will be prone to accidental BC breaks if something changes in parent class. I'd rather go for composition here to protect interface of this class.